java_console/io/src/main/java/com/rusefi/SimulatorExecHelper.java

```diff
@@ -93,9 +94,27 @@ static void destroy() {
     }
 
     public static void startSimulator() {
-        if (!new File(SIMULATOR_BINARY).exists())
-            throw new IllegalStateException(SIMULATOR_BINARY + " not found");
+        getSimulatorBinary();
+
         FileLog.MAIN.logLine("startSimulator...");
         new Thread(SimulatorExecHelper::runSimulator, "simulator process").start();
```

Comment:
can we pass the result from line 97 in to the thread, instead of doing it again in the thread?

```java
final String b = getSimulatorBinary();
new Thread(() -> runSimulator(b)...)
```